aws-dynamodb/operations.py

```diff
@@ -10,115 +10,195 @@
 logger = get_logger('aws-dynamodb')
 
 
+def remove_unwanted_param(params):
+    try:
+        params.pop('aws_region', None)
+        params.pop('assume_role', None)
+        params.pop('session_name', None)
+        params.pop('role_arn', None)
+        param_dict = {k: v for k, v in params.items() if v is not None and v != '' and v != {} and v != []}
+        return param_dict
+    except Exception as Err:
```

Comment:
remove the try- exception block to avoid the unnecessary the stack trace.

Reply:
@cs-prerna-joshi  okay, from all the functions?

Reply:
yes.

Reply:
@cs-prerna-joshi 
Done suggested changes. Please check